polaris-react/src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.tsx

```diff
@@ -26,7 +26,7 @@ const tailDownPaths = (
   <>
     <path
       d="m0 2 6.967 7.25a3 3 0 0 0 4.243.083L18.829 2h-1.442l-6.87 6.612a2 2 0 0 1-2.83-.055L1.387 2H0Z"
-      fill="#D4D4D4"
+      fill="var(--p-color-tooltip-border-tail-down-experimental)"
```

Comment:
Hard coded tooltip tail did not look right on dark surfaces. Created this new token with previous fills as the default values

<img width="271" alt="Screenshot 2024-03-28 at 2 29 17 PM" src="https://github.com/Shopify/polaris/assets/20652326/d61e68ea-6443-474b-a838-403479c8a10d">